### ISSUE TITLE
GU0031 expression body bug

### ISSUE THREAD

**Issue by tldrlol** [JohanLarsson/Gu.Analyzers#24]
The following snippet should not show a warning:

```c#
class Goof : IDisposable {
    IDisposable _disposable;
    public void Create()  => _disposable = new Disposable();
    public void Dispose() => _disposable.Dispose();
}
```

**Comment by JohanLarsson** [comment]
Thanks for reporting!
